### PR TITLE
Move opentelemetry redis example to examples repository

### PR DESCRIPTION
**Pull request by jack-berg**
Depends on first merging corresponding PR here: https://github.com/newrelic/newrelic-opentelemetry-examples/pull/608

### CODE REVIEW

File: src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practices-resources.mdx [hunk 1]
```diff
@@ -45,16 +45,16 @@ Refer to our
 [documentation and examples](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/opentelemetry-set-up-your-app)
 for monitoring service entities using OpenTelemetry.
 
-#### Required attributes
+#### Required attributes [#service-required-attributes]
 
 * [`service.name`](https://opentelemetry.io/docs/specs/semconv/attributes-registry/service): The service name you see in the UI comes from this value.
 
-#### Recommended attributes [#recommended-attributes]
+#### Recommended attributes [#service-recommended-attributes]
```
Comment (jack-berg) [inline src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practices-resources.mdx:52]: @alanwest need to differentiate between the required / recommended / entity tags sections for each entity type.

File: src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practices-resources.mdx [hunk 2]
```diff
@@ -92,6 +90,29 @@ for monitoring host entities using the host metrics receiver for the OpenTelemet
...
 
+### Redis instances [#redis]
+
+A redis instance is synthesized using data emitted from the collector [redis receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/redisreceiver). Unfortunately, there are currently no semantic conventions for redis instances, or identifying attributes in the metrics the redis receiver emits. Therefore, users are required to manually include the attributes described below.
+
+Refer to the redis example in [collector for infrastructure monitoring](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/collector-infra-monitoring/opentelemetry-collector-infra-intro/) for more details.
+
+#### Required attributes [#redis-required-attributes]
+
+* [`server.address`](https://opentelemetry.io/docs/specs/semconv/attributes-registry/server/): The redis instance name you see in the UI comes from this value.
```
Comment (jack-berg) [inline src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practices-resources.mdx:101]: See entity definition for reference: https://github.com/newrelic/entity-definitions/blame/c13e7a5d7150556c3980a64cd44047f70f3bdca0/entity-types/infra-redisinstance/definition.yml#L7-L42

**Review by ghost — APPROVED** [review]

**Comment by ally-sassman** [comment]
@jack-berg This is ready to be merged once you give the green light!

**Comment by jack-berg** [comment]
> @jack-berg This is ready to be merged once you give the green light!

Good to go!

**Pull request merged**